### ISSUE TITLE
Contributor guidelines

### ISSUE THREAD

**Issue by RayStick** [aim-rsf/Synthetic-Data#11]
- At the moment, the README links to the general contributor guidelines for the organisation. Should we make a more specific one for this repo?
- Should we include a template when someone creates a new issues or PR?
- Check what bots are active (welcome bot?) and if they need updating

**Comment by RayStick** [comment]
@BatoolMM do you have capacity to help with this issue? Some of it might not be fully necessary to work on, but interested to get your thoughts.

**Comment by BatoolMM** [comment]
Of course, happy to work on it - feel free to assign similar tasks to me!
